App/Screens/Details/Header/Header.tsx

```diff
@@ -150,7 +150,18 @@ export function Header(props: HeaderProps): React.ReactElement {
 	// FIXME Make sure the units are correctly converted.
 	const averages = pollutantAverage(api.normalized);
 
-	const time: string = formatDistanceToNow(new Date(api.pm25.date.local));
+	const time = formatDistanceToNow(new Date(api.pm25.date.local));
+
+	// FIXME The eslint keeps removing 'as string' and shot out an error.
+	const last_update_label: string = t(
+		'header.latest_update.label',
+		'Latest Update:'
+	);
+	const last_update_ago: string = t(
+		'header.latest_update.ago',
+		'{{time}} ago',
+		{ time }
+	);
```

Comment:
The eslint tool is quite trouble in here.
It keeps removing `as string` and throw error, so I have to do this to avoid it. Don't know why it is not valid.